test/misc/addresssettings/addresssettings_default_value_test.go

```diff
@@ -145,7 +153,8 @@ var _ = ginkgo.Describe("AddressSettingsDefaultValueTest", func() {
 
 		urls, err := brokerDeployer.GetExternalUrls(ExpectedURL, 0)
 		address := urls[0]
-		verifyAddressSettingsBool(address, AddressBit, "defaultLastValueQueue", true, hw)
+		value:= retrieveAddressSettings(address,AddressBit, hw)
+        gomega.Expect(value.DefaultExclusiveQueue).To(gomega.Equal(true))
```

Comment:
should `value.DefaultExclusiveQueue` be `value.DefaultExclusiveQueue.DefaultLastValueQueue` ?

Reply:
Oh, you are correct. Only matching in json struct though is LastValueQueue.